apis/network/v1alpha3/types.go

```diff
@@ -225,3 +225,249 @@ type SubnetList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Subnet `json:"items"`
 }
+
+//Azure Firewall Structs
+// +kubebuilder:object:root=true
+// A AzureFirewall is a managed resource that represents an Azure Firewall
+// +kubebuilder:printcolumn:name="READY",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
+// +kubebuilder:printcolumn:name="SYNCED",type="string",JSONPath=".status.conditions[?(@.type=='Synced')].status"
+// +kubebuilder:printcolumn:name="STATE",type="string",JSONPath=".status.state"
+// +kubebuilder:printcolumn:name="LOCATION",type="string",JSONPath=".spec.location"
+// +kubebuilder:printcolumn:name="RECLAIM-POLICY",type="string",JSONPath=".spec.reclaimPolicy"
```

Comment:
nitpick: `reclaimPolicy` is deprecated so we probably shouldn't show it here

Reply:
Removed reclaim policy declaration code.